prototype/src/main/webapp/createMeeting.js

```diff
@@ -35,3 +37,47 @@ function addMeetingGuest() {
 function removeMeetingGuest(guest) {
   document.getElementById(guest).remove();
 }
+
+/**
+ * Add a meeting time datetime-local input to the page, enclosed in 
+ * a 'li' element.
+ * The element appended to the DOM will have an id of 'meeting-time-NUMBER', 
+ * where NUMBER equal to value of MEETING_TIMES_ADDED (which is incremented
+ * each time this function is called). This is to identify the meeting time input
+ * so that users may remove the meeting time. See removeMeetingTime().
+ */
+function addMeetingTime() {
+  let listItem = document.createElement('li');
+  listItem.id = 'meeting-time-' + MEETING_TIMES_ADDED; 
```

Comment:
Did you mean `meetingTimesAdded`? Same for all the other occurences of this.

Reply:
Oh no, I forgot to change it everywhere 😅 I've changed it now, thanks for picking that up!